src/DssSpellCollateralOnboarding.sol

```diff
@@ -31,60 +35,93 @@ contract DssSpellCollateralOnboardingAction {
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmTRiQ3GqjCiRhh1ojzKzgScmSsiwQPLyjhgYSxZASQekj
     //
+    uint256 constant THREE_POINT_FIVE_PCT = 1000000001090862085746321732;
 
     // --- Math ---
+    uint256 constant THOUSAND   = 10 ** 3;
+    uint256 constant MILLION    = 10 ** 6;
+
+    // --- PRE-REQUISITE GENERAL DEPLOYS ---
+    address constant CDP_REGISTRY              = 0xBe0274664Ca7A68d6b5dF826FB3CcB7c620bADF3;
+    address constant PROXY_ACTIONS_CROPPER     = 0xa2f69F8B9B341CFE9BfBb3aaB5fe116C89C95bAF;
+    address constant PROXY_ACTIONS_END_CROPPER = 0xAa61752a5Abf86A527A09546F23FE8bCB8fAB2C4;
+    address constant CROPPER                   = 0x8377CD01a5834a6EaD3b7efb482f678f2092b77e;
+    address constant CROPPER_IMP               = 0xaFB21A0e9669cdbA539a4c91Bf6B94c5F013c0DE;
 
     // --- DEPLOYED COLLATERAL ADDRESSES ---
+    address constant ETHSTETH                  = 0x06325440D014e39736583c165C2963BA99fAf14E;
+    address constant PIP_ETHSTETH              = 0x2b465de3e69a2EC00158F0E3B4614e3582430ab2;
+    address constant MCD_JOIN_ETHSTETH_A       = 0x036A451114E3835AbEF163A67163B6B376cF2480;
```

Comment:
So it could be argued that if we added cropper's imp to changelog we should aslo add SynthetixJoinImp. Name for this is not trivial though so I'm not sure.

Reply:
Yeah been thinking on this as well. I'm not even super sold we should include the `CROPPER_IMP`.

Reply:
I think we should actually exclude the `CROPPER_IMP`. We don't store everything in the changelog such as Oracle medianizers. IMP seems like something the end user will not care about as we strive to not break the interface.